examples/openai/azure_openai.py

```diff
@@ -0,0 +1,50 @@
+import mlflow
+import openai
+import os
+import pandas as pd
+
+os.environ["OPENAI_API_KEY"] = "<YOUR AZURE OPENAI KEY>"
```

Comment:
Can we remove the key declaration in code and instead add a block comment about setting these keys through a terminal?

Reply:
+1. This can lead to leaking credentials.